### ISSUE TITLE
could you also add support for workshop deadlines?

### ISSUE THREAD

**Issue by dingran** [abhshkdz/ai-deadlines#135]
could you also add support for workshop deadlines?

**Comment by Shashi456** [comment]
Yeah Exactly, a tab could be made, while it defaults to the conference we could have a separate conference workshop tab, or in a Markdown-ish scenario, we could click on the conference to see all the workshops it has and adding the respective deadlines.

I think the latter would work out better.

**Comment by Ishaan28malik** [comment]
That I think will be there at the conference official website itself, So it is not required.
@dingran @Shashi456

**Comment by omarsar** [comment]
I think this could be really challenging to maintain especially for conferences that have many workshops. Introducing an inclusion criteria for these workshops could help but it might still involve a lot of effort to maintain. 

One option is to allow for addition of workshops as conferences, i.e., adding them separately and have an attribute that connects them to the respective conferences. This way we could list the workshops (tagged differently) on the main page and then list them under a tab in the respective conferences. The reason for this implementation is that the community can then just add individual workshops they feel are important to track as opposed to having to add all the workshops for each conference. Keeps things a bit focused and clean, and easily maintainable. Thoughts?
